keras_hub/src/layers/preprocessing/start_end_packer.py

```diff
@@ -172,10 +174,17 @@ def call(
             x = tf.concat([x, end_token_id_tensor], axis=-1)
 
         # Pad to desired length.
-        outputs = x.to_tensor(
-            default_value=self.pad_value,
-            shape=(batch_size, sequence_length),
-        )
+        if self.padding_side == "left":
```

Comment:
maybe through this into tensor_utils, we will need it for multi-segment packer as well right?